sympy/utilities/tests/test_lambdify.py

```diff
@@ -581,3 +582,9 @@ def test_issue_2790():
 def test_ITE():
     assert lambdify((x, y, z), ITE(x, y, z))(True, 5, 3) == 5
     assert lambdify((x, y, z), ITE(x, y, z))(False, 5, 3) == 3
+
+def test_10375():
+    import numpy as np
+    m = Min(x, y, z)
+    l = lambdify([x,y,z], m)
```

Comment:
`Sympy ` `Min` .I don't think we can directly map `NumPy ``amin` or `min` to `SymPy` `Min`.
[NumPy amin docs](http://docs.scipy.org/doc/numpy-1.10.0/reference/generated/numpy.amin.html) 
Its parameters are different.